internal/apiclient/policyv2.go

```diff
@@ -0,0 +1,294 @@
+package apiclient
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/infracost/infracost/internal/config"
+	"github.com/infracost/infracost/internal/schema"
+	"sort"
+)
```

Comment:
nitpick and feel free to ignore - but i like to sort the imports as follows

1. standard pkg
2. 3rd prty pkg
3. current application pkg

You can automate this in goland with the following:

<img width="963" alt="SCR-20230914-b76" src="https://github.com/infracost/infracost/assets/6455139/522a83fd-646c-4811-b9df-a7f5a8a5a85c">